### ISSUE TITLE
how to find out which checkpoint nodes were selected with memory heuristic

### ISSUE THREAD

**Issue by lexbee** [openai/gradient-checkpointing#14]
Hey guys,
Great work by the way! I was just wondering, is it possible to see which checkpoints nodes were automatically created by the memory heuristic when using keras?

**Comment by TimSalimans** [comment]
Yes! Set DEBUG_LOGGING=True in https://github.com/openai/gradient-checkpointing/blob/master/memory_saving_gradients.py#L344 and it will print the nodes that are selected as checkpoints.
